### PR TITLE
Use 302 instead of province codes for region

### PR DESCRIPTION
**Pull request by burke**
Related: https://github.com/CovidShield/server/issues/46

This'll require an update to whatever we have deployed right now too.

### CODE REVIEW

File: dev.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ up:
   - railgun
 
 env:
-  KEY_CLAIM_TOKEN: test=ON
+  KEY_CLAIM_TOKEN: test=302
```
Comment (honkfestival) [inline dev.yml:24]: Let's change `test` here to `thisisatoken` too, also in the terraform

File: test/lib/helper.rb [hunk 2]
```diff
@@ -194,7 +194,7 @@ def with_server(bin, addr, &block)
       pid = Process.spawn(
         {
           'BIND_ADDR' => addr,
-          'KEY_CLAIM_TOKEN' => 'first-token=ON:second-token=PE',
+          'KEY_CLAIM_TOKEN' => 'first-token=302:second-token=302',
```
Comment (honkfestival) [inline test/lib/helper.rb:197]: 👍

File: examples/retrieval/app.rb [hunk 3]
```diff
@@ -72,7 +72,7 @@ def maybe_fetch_new_keys
 
   def check_for_exposure
     puts "checking for exposure"
-    fetch_exposure_config("ON")
+    fetch_exposure_config('302')
```
Comment (honkfestival) [inline examples/retrieval/app.rb:75]: Exposure config will need to be expressed sub-regionally; maybe something like 302.ON? Or maybe exposure config needs to come from from a different service altogether.

Comment (burke) [reply examples/retrieval/app.rb:75]: an exposure config could have some name unrelated to region if we store it here

Comment (honkfestival) [reply examples/retrieval/app.rb:75]: Okay yeah, after discussing offline, let's go with `'302'` as you have it here

**Pull request merged**